packages/client/src/components/Modals/SavedSearchPanel.vue

```diff
@@ -73,14 +73,12 @@
                     />
                   </template>
                   <b-dropdown-item
-                    :search-id="data.item.id"
```

Comment:
The auto-fixer actually broke this. It changed the attribute name from `:searchId=` to `:search-id=`, which I guess is fine if the attribute is a component prop, but which apparently doesn't work if the attribute is going to be looked up by name:

`const searchId = e.target.getAttribute('searchid');`

Reply:
Good catch! 

I don't love anytime autofix breaks anything... but supplying a prop that the component doesn't expect just to store a value is pretty atypical usage. Thanks for the fix, that's much cleaner.